auth/cache_test.go

```diff
@@ -310,34 +409,35 @@ func TestTokenToJson_InvalidFormat(t *testing.T) {
 	}
 }
 
-func setup() (credentialDirectory, credentialPath string, err error) {
+//TODO: create testHelpers file to include testsetup and destroy
+func Setup(t *testing.T) (credentialDir, credentialPath string, err error) {
```

Comment:
I was thinking we could make a separate test helpers file, but maybe simpler for now to just specify that this is `CacheSetup()`